src/pcapi/core/bookings/validation.py

```diff
@@ -46,12 +48,12 @@ def check_quantity(offer, quantity):
         raise exceptions.QuantityIsInvalid("Vous ne pouvez réserver qu'une place pour cette offre.")
 
 
-def check_stock_is_bookable(stock):
+def check_stock_is_bookable(stock: Stock) -> None:
     if not stock.isBookable:
         raise exceptions.StockIsNotBookable()
 
 
-def check_expenses_limits(expenses, requested_amount, offer):
+def check_expenses_limits(expenses, requested_amount: int, offer: Offer) -> None:
```

Comment:
```suggestion
def check_expenses_limits(expenses, requested_amount: Decimal, offer: Offer) -> None:
```
(avec l'import qui va bien)